packages/augur-api/src/api/Contracts.ts

```diff
@@ -1,12 +1,29 @@
 import { GenericAugurInterfaces } from "@augurproject/core";
 import { ContractAddresses } from "@augurproject/artifacts";
 
+export type SomeRepToken<TBigNumber> = GenericAugurInterfaces.ReputationToken<TBigNumber> | GenericAugurInterfaces.TestNetReputationToken<TBigNumber>;
+
 export class Contracts<TBigNumber> {
   public augur: GenericAugurInterfaces.Augur<TBigNumber>;
   public universe: GenericAugurInterfaces.Universe<TBigNumber>;
+  public cash: GenericAugurInterfaces.Cash<TBigNumber>;
+  public reputationToken: SomeRepToken<TBigNumber> | null = null;
+  private readonly dependencies: GenericAugurInterfaces.Dependencies<TBigNumber>;
 
   public constructor (addresses: ContractAddresses, dependencies: GenericAugurInterfaces.Dependencies<TBigNumber>) {
+    this.dependencies = dependencies;
     this.augur = new GenericAugurInterfaces.Augur<TBigNumber>(dependencies, addresses.Augur);
     this.universe = new GenericAugurInterfaces.Universe<TBigNumber>(dependencies, addresses.Universe);
+    this.cash = new GenericAugurInterfaces.Cash<TBigNumber>(dependencies, addresses.Cash);
+  }
+
+  public async setReputationToken(testnet: boolean = false) {
```

Comment:
I don't call this in the `Augur.ts` file's `create(...)` method. I'm not sure what the best approach would be:
1. Always create the production rep token but call `setReputationToken(true)` later for tests or
2. Add `testnet: bool` argument to `Augur.create(...)`.

Reply:
You can base this off of the network id. Any network id other than 1 is a testnet.

For the mainnet test deployment we actually wont have the faucet, but the REP contract will point to a fake, previously deployed legacy REP contract so this will work in that case as well.

Reply:
Thanks!